.github/workflows/release.yml

```
@@ -0,0 +1,71 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    branches: [main, staging]
```

actionlint (1.7.7)

expected "inputs" key for "workflow_dispatch" section but got "branches"

(syntax-check)